### PR TITLE
Organisation Page

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Review by techsmyth — APPROVED** [review]
Nice to have this page there! 

Overall working nicely. Only comment is that I think that what you were first doing, having the owners / admins up in the top section under the tags would be a much better usage of space. The members can be a section below - but as there are in general not likely to be more than 1-2 owners / admins then I think these should go to the top, also so that people know who to connect with. 

But this we can pick up in a future refinement when we can look at the whole page UX.

**Review by atstojanov — CHANGES_REQUESTED** [review]

File: src/routing/routing.tsx [hunk 1]
```diff
@@ -42,9 +44,11 @@ export const Routing: FC = () => {
       <RestrictedRoute path="/user">
         <div>User Page: Comming Soon!</div>
       </RestrictedRoute>
-      <RestrictedRoute path="/organisation">
-        <div>Organisation Page: Comming Soon!</div>
-      </RestrictedRoute>
+      <Route path="/organisation/:organisationId">
```
[the remark this answers is not in the document]
Comment (techsmyth) [reply src/routing/routing.tsx:47]: We need a discussion I think on route naming, as I also doubt that we need to have 'challenges' in the main route.

[the remark this answers is not in the document]
Comment (atstojanov) [reply src/routing/routing.tsx:47]: We need fundamental refactoring of the routing, but for that the UI/UX design update is required.

[the remark this answers is not in the document]
Comment (techsmyth) [reply src/routing/routing.tsx:47]: I suggest for now just switch to the american version singular i.e. /organization/

File: src/i18n/en/translation.en.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
     "alternativesubheader": "Need help? Find relevant users, organisations, groups and not least Challenges!",
     "body": "",
     "organization": {
-      "no-tags": "No tags available",
```
Comment (atstojanov) [inline src/i18n/en/translation.en.json:18]: this one is used in `OrganizationPopUp.tsx`

image

**Review by atstojanov — CHANGES_REQUESTED** [review]

**Review by atstojanov — APPROVED** [review]

**Pull request merged**